### ISSUE TITLE
Audio normalization in train/val/test

### ISSUE THREAD

**Issue by thanhtvt** [hkchengrex/MMAudio#78]
Hi @hkchengrex,

Congratulations on your great work! I really enjoyed reading your paper and running your well-organized codebase. I noticed one thing: in the snippet below, it seems that you're **normalizing the audio from the training set but not from the validation set**. I understand why this might be the case for the test set, since it's not used for generation/evaluation anyway, but could you clarify why the validation set is treated differently?

https://github.com/hkchengrex/MMAudio/blob/ec6ab44928dcca2df51e5894140ad140149c3aa3/training/extract_video_training_latents.py#L54-L75

**Comment by thanhtvt** [comment]
When I followed your settings and trained the model on VGGSound only, I couldn’t generate meaningful audio—the output was very noisy. I also noticed that the model barely generalizes to the validation set: the loss decreases only slightly before rising again quite rapidly (see attached). Could you provide some guidance on this? I kept the training settings the same, except that I reduced the number of training steps to 150k and set the learning rate schedule to [100k, 120k].

<img width="1549" height="869" alt="Image" src="https://github.com/user-attachments/assets/2fcda768-6513-415f-bc41-11b2a009dfcc" />

**Comment by hkchengrex** [comment]
For reference, you can take a look at this issue: https://github.com/hkchengrex/MMAudio/issues/49

If you don't train on the audio-text datasets, it overfits fairly quickly. We apply early stop as mentioned in the paper (by looking at the validation performance).

**Comment by thanhtvt** [comment]
Thanks for the quick reply! Issue #49 discusses training the 44k-large version, while I’m currently using the 16k-small one. Are there any differences in the loss curves between the two?
